README.rst

```diff
@@ -21,23 +23,98 @@ pythx
+
+        Commands:
+        check    Submit a new analysis job based on source code, byte code, or...
+        login    Login to your MythX account
+        logout   Log out of your MythX account
+        openapi  Get the OpenAPI spec in HTML or YAML format
+        ps       Get a greppable overview of submitted analyses
+        refresh  Refresh your MythX API token
+        report   Check the detected issues of a finished analysis job
+        status   Get the status of an analysis by its UUID
+        top      Display the most recent analysis jobs and their status
+        version  Print version information of PythX and the API
+
+
+By default, PythX comes enabled with a trial user:
+
+.. code-block::
+
+        $ pythx login
+        Please enter your Ethereum address [0x0000000000000000000000000000000000000000]:
+        Please enter your MythX password [trial]:
+        Successfully logged in as 0x0000000000000000000000000000000000000000
+
+
+Submit an Solidity source file for analysis:
+
+.. code-block::
+        $ pythx check -sf test.sol
+        Analysis submitted as job df137587-7fc1-466a-a4b2-d63392099682
+
+
+Check the status of your analysis job:
 
-A Python library for the MythX platform
+.. code-block::
+        $ pythx status df137587-7fc1-466a-a4b2-d63392099682
+        ╒════════════════╤══════════════════════════════════════╕
+        │ uuid           │ df137587-7fc1-466a-a4b2-d63392099682 │
+        ├────────────────┼──────────────────────────────────────┤
+        │ apiVersion     │ v1.4.3                               │
+        ├────────────────┼──────────────────────────────────────┤
+        │ mythrilVersion │ 0.20.0                               │
+        ├────────────────┼──────────────────────────────────────┤
+        │ maestroVersion │ 1.2.3                                │
+        ├────────────────┼──────────────────────────────────────┤
+        │ harveyVersion  │ 0.0.13                               │
+        ├────────────────┼──────────────────────────────────────┤
+        │ maruVersion    │ 0.3.4                                │
+        ├────────────────┼──────────────────────────────────────┤
+        │ queueTime      │ 0                                    │
+        ├────────────────┼──────────────────────────────────────┤
+        │ runTime        │ 0                                    │
+        ├────────────────┼──────────────────────────────────────┤
+        │ status         │ Finished                             │
+        ├────────────────┼──────────────────────────────────────┤
+        │ submittedAt    │ 2019-03-05T10:24:05.071Z             │
+        ├────────────────┼──────────────────────────────────────┤
+        │ submittedBy    │ 123456789012345678901234             │
+        ╘════════════════╧══════════════════════════════════════╛
 
 
-* Free software: MIT license
-* Documentation: https://pythx.readthedocs.io.
+Get the analysis report. Resolving the source maps is still a bit buggy,
```

Comment:
This is just my opinion but saying `resolving the source maps` is a bit jargony (had to quickly google what it meant :stuck_out_tongue: ).
It would be more straightforward to say: 
`Get the analysis report. Pinpointing the line and column locations is still a bit buggy,
sorry. :)`

Reply:
Good catch :)
efbe065b5df3a45797e26f65c1c1896016b93ab3